docs/Actions.md

```diff
@@ -1746,6 +1746,23 @@ It can process unit test results if formatted as junit report as shown in [xctes
 
 ## Misc
 
+### twitter
+
+Post a tweet on twitter.
+requires you to setup an app on twitter.com and obtain consumer and access_token
+
+
```

Comment:
I know that these don't render in the HTML, but would you mind just cleaning up at least one of these spaces? :smile: